packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -34,13 +38,24 @@ <h2>Meetings</h2>
 
     <section>
       <h2>Authentication</h2>
-      <p>[[AUTHENTICATION DESCRIPTION]]</p>
+
+      <p>This section contains all of the logic needed to fully authenticate and prepare the Webex JS SDK for meetings usage. There are two steps that need to be performed before the meetings functionality of the JS SDK is available for use:</p>
+      <ul>
+        <li>Webex.init() => Initialize the Webex JS SDK</li>
+        <li>webex.meetings.register() => Register the local device using</li>
+      </ul>
+      <p>Once the Webex JS SDK instance is ready and registered, The rest of the meetings functionality of the Webex JS SDK becomes available. These values can be confirmed by checking:</p>
+      <ul>
+        <li>webex.ready => Determine if the webex instance is ready</li>
+        <li>webex.meetings.registered => Determine if the webex instances local device is registered</li>
+      </ul>
```

Comment:
This is too wordy and isn't in the order of expected execution
This can be a bit troublesome and confusion, however the explanation itself is needed to explain what the api does

I suggest adding simple collapsible add the end of [each] section that has a simple example on how to implement, like how a lot of CSS frameworks have in their docs

```js
let webex = window.webex = Webex.init({
    credentials: {
      access_token: tokenElm.value
    }
  });

  webex.once('ready', () => {
    webex.meetings.register()
    .then(() => {
      console.log('Authentication#register() :: successfully registered');
    })
    .catch((error) => {
      console.warn('Authentication#register() :: error registering', error);
    })
  });
```

Reply:
Cleaning this up and leaving it as is for now, a later PR will add this feature/documentation.